certified-connectors/Rainbird/apiDefinition.swagger.json

```diff
@@ -0,0 +1,865 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.1.0",
+    "title": "Rainbird",
+    "description": "Rainbird is a no-code intelligent automation platform that enables you to automate complex decisions. This connector enables you to interact with knowledge maps you have built in Rainbird.",
+    "contact": {
+      "name": "Rainbird Technologies Ltd",
+      "url": "https://rainbird.ai/contact-us/",
+      "email": "support@rainbird.zohodesk.eu"
+    }
+  },
+  "host": "enterprise-api.rainbird.ai",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/start/{kmID}": {
+      "get": {
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "environment",
+            "type": "string",
+            "enum": [
+              "enterprise-api.rainbird.ai",
+              "api.rainbird.ai"
+            ],
+            "required": true,
+            "description": "Account environment: Community (api.rainbird.ai) or Enterprise (enterprise-api.rainbird.ai).",
+            "x-ms-summary": "Environment",
+            "x-ms-visibility": "important"
+          },
+          {
+            "description": "The knowledge map ID to start a session against",
+            "in": "path",
+            "name": "kmID",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Knowledge Map ID (kmId)",
+            "x-ms-url-encoding": "single",
+            "x-ms-visibility": "important"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Session start information",
```

Comment:
@Mat-Rig can you please provide more detailed descriptions for response codes? You can refer to [this example](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).

Reply:
I added more codes and associated description, what do you think?